e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -14,6 +14,7 @@ export function visitAlias(alias) {
 
 /**
  * Open native (SQL) editor and alias it.
+ * @deprecated To avoid typing SQL, which is unreliable, use the helper startNewNativeQuestion
```

Comment:
We still need to type. The main difference is that this helper goes through the UI flow (which makes it slower) so when it lands on the native editor page it still waits for some requests to resolve. Mainly the selection of the last used database (this causes a re-render).

OTOH, using the programatic `startNewNativeQuestion` gives you the option to bake in the database you want to use. This is what is making all these tests flaky.

Reply:
I stand corrected - we don't _have to_ type because `startNewNativeQuestion` accepts the `query`. It should be used like that only when we want to test the **ad-hoc** native question.

Even if we use this helper in combo with the actual `cy.type` or `realType`, it will still significantly reduce the flakiness over the `openNativeEditor`.